vscode/README.md

```diff
@@ -8,6 +8,16 @@ The Toit extension adds language support for the Toit programming language, incl
 
 ## Quick start
 
+### toitlang.org
+The open source version of Toit provides syntax highlighting and the
+language server (providing completions, diagnostics, ...). The easiest way to set it up is to
```

Comment:
I'd drop the parenthesis here and write:

... and the language server with completions, diagnostics, etc.